fe/fe-core/src/main/java/org/apache/doris/catalog/HdfsStorageVault.java

```diff
@@ -85,17 +90,66 @@ public HdfsStorageVault(String name, boolean ifNotExists, boolean setAsDefault)
     }
 
     @Override
-    public void modifyProperties(Map<String, String> properties) throws DdlException {
-        for (Map.Entry<String, String> kv : properties.entrySet()) {
+    public void modifyProperties(Map<String, String> newProperties) throws DdlException {
+        for (Map.Entry<String, String> kv : newProperties.entrySet()) {
             replaceIfEffectiveValue(this.properties, kv.getKey(), kv.getValue());
         }
+        checkConnectivity(this.properties);
```

Comment:
I suggest to create a new interface to call this `checkConnectivity`, not put it in `modifyProperties`.
Because other developer may call this method in replay logic, for example, and it is very error prone

Reply:
And I think this method is also useful for all other feature which need to connect to hdfs, how about make it as a general purpose checking method?

Reply:
The method only used in hdfsStorageVault, i don't want to take effects with other modules such as resource, maybe in the future we can move the mothod into ObjStorage.java if need